GetandTake/Configuration/CustomExceptionHandler.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.AspNetCore.Diagnostics;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace GetandTake.Configuration;
+
+public static class CustomExceptionHandler
+{
+    public static void ExceptionHandler(this WebApplication app)
+    {
+        app.UseExceptionHandler(exceptionHandlerApp =>
+        {
+            exceptionHandlerApp.Run(async context =>
+            {
+                context.Response.StatusCode = StatusCodes.Status500InternalServerError;
+                context.Response.ContentType = Text.Plain;
+
+                await context.Response.WriteAsync("An exception was thrown.");
+
+                var exceptionHandlerPathFeature =
+                    context.Features.Get<IExceptionHandlerPathFeature>();
+                if (exceptionHandlerPathFeature != null)
+                {
```

Comment:
Guess this is going to be a big nesting if you will add more condition for internal logic check. Let's revert as I showed you before:
```suggestion
                if (exceptionHandlerPathFeature == null)
                {
                    return;
                 }
```

It is a really good practice to try to avoid big nested items as much as possible in all languages.